config/env_sample.json

```diff
@@ -93,19 +93,19 @@
     "/*  LTI is disabled by default": "*/",
     "STUDENT_DASHBOARD_LTI": false,
     "/*  LTI 1.3 configuration":"*/",
-    "/*  The first key of LTI_CONFIG is your Canvas domain name":"*/",
+    "/*  The first key of LTI_CONFIG is your environment specific Canvas url and not domain specific":"*/",
```

Comment:
Is this so?  If the `canvas*.instructure.com` URLs are used for launch, then to MyLA it appears to have be running there?  It doesn't redirect to a domain-specific URL?

Reply:
The issuer id is always independent of the domain specific and canvas is strict on that. The LTI launch will fail if you use domain specific URL. I hope that answers your Question